### DISCUSSION TITLE
Possible little errors in tolerances

### DISCUSSION THREAD

**Discussion by RuiheLi** [pybamm-team/PyBaMM#2389] [General]
Dear PyBaMM,

I just found that in Line 154 in /pybamm/parameters/lithium_ion_parameters.py, "D_e__c_e" is used to set tolerances for kappa_e_dimensional. Would it be possible that it is "kappa_e__c_e"? Though they are both 10. 

Thanks!

**Comment by valentinsulzer** [comment]
Yes, that's a bug, can you open a PR to fix?

**Comment by RuiheLi** [comment]
No problem. Will do later. (After I figure that out with Simon)
